recipes/tabulate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tabulate" %}
+{% set version = " v1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/p-ranav/tabulate/archive/{{ version }}.tar.gz
```

Comment:
here you should have `https://github.com/p-ranav/tabulate/archive/v{{ version }}.tar.gz` (so add a `v` in front of `{{ version }}`.